### DISCUSSION TITLE
Laravel Sail configuration.

### DISCUSSION THREAD

**Discussion by QSoto** [recca0120/vscode-phpunit#165] [Q&A] [answered]
Hey Folks! 
I've been trying to set up the Test Explorer to run over Laravel Sail, no success yet. Any help would be useful.

This is how my `settings.json` looks like right now:

```
    "phpunit.command": "./vendor/bin/sail",
    "phpunit.php": "php",
    "phpunit.phpunit": "vendor/bin/phpunit",
    "phpunit.args": [
        "-c",
        "phpunit.xml"
    ],
    "phpunit.paths": {

        // ${pwd} (current directory) is mounted to /app
        "${workspaceFolder}": ".",
        // ${workspaceFolder} is mapped to /app
    },
```

This is the output command:
`./vendor/bin/sail php vendor/bin/phpunit '--configuration=phpunit.xml' '--teamcity' '--colors=never'`

It's weird, the command looks right, but when I try to use the GUI the output is: 
![image](https://github.com/recca0120/vscode-phpunit/assets/18690766/164273c0-778d-4aae-9b37-21dc6ac97e02)

_Running over Ubuntu 22.04, php8.1_

**Comment by andriawan** [comment, marked as answer]
try this configuration as mentioned on documentation

```{
  "phpunit.command": "docker compose exec -u sail laravel.test ${php} ${phpargs} ${phpunit} ${phpunitargs}",
  "phpunit.phpunit": "artisan test",
  "phpunit.paths": {
    "${workspaceFolder}": "/var/www/html",
  }
}
